cvfy.py

```diff
@@ -223,7 +236,7 @@ def getTextArray():
         pass
     return (textdata)
 
-def getImageArray():
+def getImageArray(mode='numpy_array'):
```

Comment:
Default mode should be `file_path` instead - since it is more generic of the two.